### DISCUSSION TITLE
Unlock charger cable

### DISCUSSION THREAD

**Discussion by erik-131** [alandtse/tesla#337] [General]
Am I missing something, or are there not any way to unlock the cable lock on the car itself? I have to open the app to do this, or enter the car and use the dashboard.

**Comment by anhaff** [comment]
Charging equipment from Tesla has a button on the plug you push into the charging port. When you press the button while the car is unlocked, plug/cable is released.
I have seen also one non-Tesla charging cable that has this button.

**Reply by erik-131** [reply]
Unfortunately I don't have that button on the cable

**Comment by mafelipe16** [comment]
In previous versions was available an unlock charger lock but in this current version has dissappear, I used it to unlock it via an automation.

**Comment by samsmooth** [comment]
+1

**Comment by keldbroe** [comment]
Yea I used that feature a lot in previous versions. Had a Ikea button located close by my car running an automation releasing the cable.

**Comment by erik-131** [comment]
It has been added in the new release, works perfectly! Just update in HACS

**Reply by keldbroe** [reply]
Damn.. Didn't check before my post. Thanks a lot!
